spec/abstract-jobs.html

```diff
@@ -111,24 +138,10 @@ <h1>Execution</h1>
 
       <ul>
         <li>
-          If the following conditions about an ECMAScript object are met, then
-          the implementation may replace each reference to the object from the
-          [[Target]] field of a FinalizationGroup's [[Cells]] List, or the
-          [[Target]] internal slot of a WeakRef, to the value ~empty~:
-          <ul>
-            <li>The object is not included in any agent's [[KeptAlive]] List.</li>
-            <li>
-                No possible future execution of the program would observably
-                reference the object, except through a chain of references which
-                includes a [[Target]] field of a FinalizationGroup's [[Cells]] List,
-                or the [[Target]] internal slot of a WeakRef.
-              </p>
-              <p>
-                NOTE: The exact guarantee remains under discussion in
-                <a href="https://github.com/tc39/proposal-weakrefs/issues/115">#115</a>.
-              </p>
-            </li>
-          </ul>
+          If an object is not live, then the implementation may replace any
```

Comment:
can “live” here link to the liveness section?

Reply:
Doesn't it autolink due to the dfn?

Reply:
oh hmm, probably so.